.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
I think it is a good idea to just allow this? Do we really want to go full JIRA on our users? 

(Background: JIRA is very hated because in most companies a lot customization is done with many custom fields and templates which make it very bureaucratic to use)